lib/inspector.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const connect = process.binding('inspector').connect;
+const EventEmitter = require('events');
+
+if (!connect)
+  throw new Error('Inspector is not available');
+
+const onMessageSymbol = Symbol('onMessage');
+
+class Session extends EventEmitter {
+  constructor() {
+    super();
+    this._connection = connect((message) => this[onMessageSymbol](message));
+    this._nextId = 1;
+    this._messageCallbacks = new Map();
+  }
+
+  [onMessageSymbol](message) {
+    const parsed = JSON.parse(message);
+    if (parsed['id']) {
+      var callback = this._messageCallbacks.get(parsed['id']);
+      if (callback)
+        callback(parsed['result'] || null, parsed['error'] || null);
+      this._messageCallbacks.delete(parsed['id']);
+    } else {
+      this.emit(parsed['method'], parsed);
+      this.emit('protocolNotification', parsed);
+    }
+  }
+
+  post(method, params, callback) {
+    if (!this._connection)
+      throw new Error('Session is not connected');
+    const id = this._nextId++;
```

Comment:
IDs are only alive for the duration of a `Session`, right? In other words, if I open and close one session, and then open another session, is it guaranteed that the IDs from the two sessions will not conflict?